docs/troubleshooting/github-connection-cannot-be-removed.mdx

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_label: Github connection can't be removed
```

Comment:
Let's make the sidebar label "Connection can't be removed" since the word "GitHub" will overflow the title to two rows